### PR TITLE
Feature/account menu

### PR DESCRIPTION
**Pull request by ektasawant**
Fixes PXBLUE-2423 .

#### Changes proposed in this Pull Request:
-  App bar Desktop view
- App bar Mobile view
- Chip component which opens user menu.

Note: Mobile view for user chip menu is remaining.

#### To Test:
- http://localhost:4200/account-menu/in-an-app-bar

### CODE REVIEW

File: src/app/pages/account-menu/in-an-app-bar/in-an-app-bar.component.html [hunk 1]
```diff
@@ -0,0 +1,171 @@
...
+                    <mat-nav-list [style.paddingTop.px]="0">
+                        <pxb-info-list-item *ngFor="let item of items2" [dense]="true" (click)="open4 = false">
+                            <mat-icon pxb-icon>{{ item.icon }}</mat-icon>
+                            <div pxb-title>{{ item.title }}</div>
+                        </pxb-info-list-item>
+                    </mat-nav-list>
+                </div>
+            </pxb-user-menu>
+        </mat-toolbar>
+        <!-- Text Menu Example -->
```
Comment (ektasawant) [inline src/app/pages/account-menu/in-an-app-bar/in-an-app-bar.component.html:142]: Commented last user menu variation.

**Comment by emclaug2** [comment]
Can you move the `chip` folder into the `in-an-app-bar` folder since they should be together?

File: src/app/pages/account-menu/chip/chip.component.html [hunk 2]
```diff
@@ -0,0 +1,11 @@
+<mat-chip-list class="pxb-chip-list">
+    <mat-chip class="pxb-chip" [ngClass]="selected ? 'highlight' : ''" (click)="toggleChip()">
```
Comment (emclaug2) [inline src/app/pages/account-menu/chip/chip.component.html:2]: `[class.highlight]="selected"` reads cleaner

File: src/app/pages/account-menu/chip/chip.component.ts [hunk 3]
```diff
@@ -0,0 +1,26 @@
...
+export class ChipComponent implements OnInit {
+    @Input() icon: string;
+    @Input() label: string;
+    @Input() selected = false;
+    @Output()
+    openUserMenu = new EventEmitter<boolean>();
+    open: false;
+    constructor() {}
+
+    ngOnInit(): void {}
```
Comment (emclaug2) [inline src/app/pages/account-menu/chip/chip.component.ts:17]: ngOnInit can be removed.

**Review by emclaug2 — APPROVED** [review]
This looks good to me, just see few small comments.

I know the chip example is missing but this is totally fine.  Could you make a feature request to add it in the future?  It cannot be added until v6.0.0 of @pxblue/angular-components is published with a fix for the UserMenu bottomsheet manual state management.

**Comment by ektasawant** [comment]
> This looks good to me, just see few small comments.
> 
> I know the chip example is missing but this is totally fine. Could you make a feature request to add it in the future? It cannot be added until v6.0.0 of @pxblue/angular-components is published with a fix for the UserMenu bottomsheet manual state management.

How to make feature request? Could you please guide me?

**Comment by emclaug2** [comment]
> > This looks good to me, just see few small comments.
> > I know the chip example is missing but this is totally fine. Could you make a feature request to add it in the future? It cannot be added until v6.0.0 of @pxblue/angular-components is published with a fix for the UserMenu bottomsheet manual state management.
> 
> How to make feature request? Could you please guide me?

Try this link: https://github.com/pxblue/angular-design-patterns/issues/new/choose

**Comment by ektasawant** [comment]
> > > This looks good to me, just see few small comments.
> > > I know the chip example is missing but this is totally fine. Could you make a feature request to add it in the future? It cannot be added until v6.0.0 of @pxblue/angular-components is published with a fix for the UserMenu bottomsheet manual state management.
> > 
> > 
> > How to make feature request? Could you please guide me?
> 
> Try this link: https://github.com/pxblue/angular-design-patterns/issues/new/choose

Done [142](https://github.com/pxblue/angular-design-patterns/issues/142). Thanks!!!

**Pull request merged**